libs/components/tiles/src/lib/modules/tiles/tile-dashboard/tile-dashboard.component.ts

```diff
@@ -51,7 +51,7 @@ export class SkyTileDashboardComponent implements AfterViewInit, OnDestroy {
   }
 
   /**
-   * Provides an observable to send commands to the tile dashboard. The commands should respect the
+   * An observable to send commands to the tile dashboard. The commands should respect the
```

Comment:
```suggestion
   * An observable to send commands to the tile dashboard. The commands must respect the
```
Unrelated to the other changes we're making, but I think we should change "should" to "must" to avoid the possibility that this reads like a suggestion rather than a requirement.